mne/viz/_brain/_brain.py

```diff
@@ -2456,6 +2456,25 @@ def set_time_point(self, time_idx):
         self._data['time_idx'] = time_idx
         self._update()
 
+    def set_time(self, time_sec):
+        """Set the time to display (in seconds).
+
+        Parameters
+        ----------
+        time_sec : float
+            The time to show.
```

Comment:
I wonder if it would be more future-proof to call the param `time` or `t` or so, in case we decide to add a `(time)unit` kwarg later?

Reply:
I would also call it time for api consistency

Reply:
And say it docstring it should be seconds